Shoko.Server/API/SignalR/Models/FileHashedEventSignalRModel.cs

```diff
@@ -0,0 +1,11 @@
+using Shoko.Plugin.Abstractions;
+
+namespace Shoko.Server.API.SignalR.Models
+{
+    public class FileHashedEventSignalRModel : FileEventSignalRModel
+    {
+        public FileHashedEventSignalRModel(FileHashedEventArgs eventArgs) : base(eventArgs)
+        {
+        }
+    }
+}
```

Comment:
Unless this is adding some sort of value, it is probably worth just passing in the base class.

Reply:
We can add fields to it in the future if needed. Yes we can change it.

Reply:
Yeah, I would just add it when it's needed, not trying to plan too far in the future